apps/web/src/pages/Pools/PoolAdvertisementPage/PoolAdvertisementPage.tsx

```diff
@@ -127,6 +131,93 @@ const DeadlineDialogReturn = ({
   return null;
 };
 
+const buildWhoCanApplyText = (
+  areaOfSelection: PoolAreaOfSelection | null | undefined,
```

Comment:
That's true, but this isn't a component.  You can see the result is just rendered on the page between brace brackets instead of as a component like `<WhoCanApplyText prop1="" prop2=""/>`

![image](https://github.com/user-attachments/assets/69d3628c-fe85-4d74-99c0-bc2e897c3308)

You're probably right - it should be a component.  I did that here: [Factor out who can apply to component](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11446/commits/d136214f19c6314b152ad5f426db4343742890d1)